### PR TITLE
[PM-4925] Implemented getPublicKey to follow WSC standards

### PR DESCRIPTION
**Pull request by gbubemismith**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [X] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective
This fixes the issue raised https://github.com/bitwarden/clients/issues/6913 in adhering to W3C standards.

## Code changes

- **libs/common/src/vault/services/fido2/fido2-authenticator.service.ts:** Added a new variable `pubKeyDer` that returns the public key in DER SubjectPublicKeyInfo format.
- **apps/browser/src/vault/fido2/webauthn-utils.ts** The `getPublicKey` function is updated to return in public key in DER SubjectPublicKeyInfo format.

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Comment by coroiu** [comment]
I don't think SPKI is the correct key format

File: libs/common/src/vault/services/fido2/fido2-authenticator.service.ts [hunk 1]
```diff
@@ -126,7 +127,7 @@ export class Fido2AuthenticatorService implements Fido2AuthenticatorServiceAbstr
 
       try {
         keyPair = await createKeyPair();
-
+        pubKeyDer = await crypto.subtle.exportKey("spki", keyPair.publicKey);
```
Comment (coroiu) [inline libs/common/src/vault/services/fido2/fido2-authenticator.service.ts:130]: issue: this is the wrong format. The key needs to be returned in `DER` format, a conversion we don't currently support. To merge this PR we need to implement that conversion and then also find a way to validate that the data we return can actually be read as `DER`, maybe using external applications that can read `DER`

**Review by coroiu — CHANGES_REQUESTED** [review]

File: libs/common/src/vault/services/fido2/fido2-authenticator.service.ts (hunk 1 above)
Comment (coroiu) [reply libs/common/src/vault/services/fido2/fido2-authenticator.service.ts:130]: I can't approve before you publish the PR :)

Comment (gbubemismith) [reply libs/common/src/vault/services/fido2/fido2-authenticator.service.ts:130]: Well, it is not really the same thing, but when using the `WebCryptoAPI` for exporting the `SPKI, ` it's encoded using DER.

**Review by coroiu — APPROVED** [review]
Confirmed that this seems to be exporting the same format as chrome, at least according to https://lapo.it/asn1js/

**Pull request merged**